.github/workflows/benchmarks.yml

```diff
@@ -67,3 +68,65 @@ jobs:
       - name: Run MT benchmarks
         working-directory: ${{github.workspace}}/build
         run: ./benchmark/multithread_bench
+
+  benchmarks-windows:
+    name: Windows
+    env:
+      BUILD_DIR : "${{github.workspace}}/build/"
+      HWLOC_PATH: "${{github.workspace}}/build/vcpkg/packages/hwloc_x64-windows"
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ['windows-2022']
```

Comment:
This should be hardcoded in-place as it is used only in one place and this variable doesn't have any other values than `windows-2022`.

Reply:
Done